tools/dts/mpfs_icicle.dts

```diff
@@ -0,0 +1,474 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
```

Comment:
This looks to be the offending SPDX tag. The SPDX syntax changed a few versions ago, I'd suggest to replace:
```suggestion
// SPDX-License-Identifier: (GPL-2.0-only OR MIT)
```
(If the original authors released it under `GPL-2.0` without clarification, it is fine for us to to pick one of the options).

Reply:
Yeah I grabbed this from a series of yocto patches for their bsp. I'll make the change